src/assets/styles/pages/_resources.scss

```diff
@@ -0,0 +1,388 @@
+
+    // Override the default style defined in _base.scss
+    ul {
+      margin: rem(16) 0 rem(16) 0;
+
+      li {
+        margin-bottom: rem(8);
+
+        &::before {
+          content: none;
+        }
+      }
+    }
+
+    // Custom checkboxes. Reference: https://codepen.io/spacemonkey/pen/vmZROv
+    .filter-checkbox {
+      opacity: 0; // hide it
+      position: absolute; // take it out of document flow
+
+      & + label {
+        cursor: pointer;
+        padding: 0;
+        position: relative;
+      }
+
+      // Box
+      & + label::before {
+        border-radius: rem(3);
+        content: '';
+        display: inline-block;
+        height: rem(20);
+        margin: rem(4) rem(10) 0 0;
+        outline: rem(1) solid var(--fl-fgColor, var(--indigo-500));
+        vertical-align: text-top;
+        width: rem(20);
+      }
+
+      // Checkmark
+      &:checked + label::after {
+        color: var(--fl-bgColor, $white);
+        content: '\2713';
+        display: inline-block;
+        font-size: rem(18);
+        height: rem(2);
+        left: rem(4);
+        position: absolute;
+        top: 0;
+        width: rem(2);
+      }
+    }
+
+    // End of custom checkboxes
+  }
+
+  .apply-button {
+    background-color: var(--fl-fgColor, var(--indigo-500));
+    border: 2px solid var(--fl-fgColor, var(--indigo-500));
+    border-radius: rem(5);
+    color: var(--fl-bgColor, $white);
+    font-size: rem(16);
+    font-weight: var(--fw-medium);
+    line-height: rem(24);
+    margin-top: rem(16);
+    padding: rem(10) rem(14);
+
+    &:hover {
+      background-color: var(--fl-fgColor, var(--indigo-800)) !important;
+      border: 2px solid var(--fl-fgColor, var(--indigo-800)) !important;
+      color: var(--fl-bgColor, #fff) !important;
+    }
+
+    &:focus {
+      background-color: var(--fl-fgColor, var(--indigo-800));
+      border: 2px solid var(--fl-fgColor, var(--indigo-800));
+      color: var(--fl-bgColor, #fff);
+      outline: var(--fl-fgColor, var(--indigo-500)) solid rem(3);
+      outline-offset: rem(3);
+    }
+  }
+
+  .reset-button {
+    background-color: var(--fl-fgColor, var(--grey-100));
+    border: 2px solid var(--fl-fgColor, var(--indigo-500));
+    border-radius: rem(5);
+    color: var(--fl-bgColor, var(--indigo-500));
+    font-size: rem(16);
+    font-weight: var(--fw-medium);
+    padding: rem(10) rem(14);
+
+    &:hover {
+      background-color: var(--fl-fgColor, var(--indigo-800)) !important;
+      border: 2px solid var(--fl-fgColor, var(--indigo-800)) !important;
+      color: var(--fl-bgColor, #fff) !important;
+    }
+
+    &:focus {
+      background-color: var(--fl-fgColor, var(--indigo-800));
+      border: 2px solid var(--fl-fgColor, var(--indigo-800));
+      color: var(--fl-bgColor, #fff);
+      outline: var(--fl-fgColor, var(--indigo-500)) solid rem(3);
+      outline-offset: rem(3);
+    }
+  }
+
+  .resources-no-results {
+    background-color: #f2f2f2;
+    font-size: 22px;
```

Comment:
use `rem()` function instead of `px`. `px` should only be used for very small px values such as borders.

This comment applies to all css.